pbp.sh

```diff
@@ -1,27 +1,14 @@
-echo "$(tput setaf 5)Pulling and bundling consumer-layouts...$(tput sgr0)"
-cd consumer-layouts
-git pull
-bundle install
+#!/usr/bin/env bash
```

Comment:
Basically what you said. It's a hint to the environment about what bash binary to use.